lib/archethic/contracts/interpreter/action_interpreter.ex

```diff
@@ -13,13 +13,13 @@ defmodule Archethic.Contracts.Interpreter.ActionInterpreter do
   @doc """
   Parse the given node and return the trigger and the actions block.
   """
-  @spec parse(any()) :: {:ok, atom(), any()} | {:error, any(), String.t()}
-  def parse({{:atom, "actions"}, _, [keyword, [do: block]]}) do
+  @spec parse(any(), list) :: {:ok, atom(), any()} | {:error, any(), String.t()}
```

Comment:
You should define a `@type function_key() :: {String.t(), integer()}` somewhere and use it in the specs (`list(function_key())`)